### PR TITLE
Adds Admin-Only Nanotrasen Commander Headset and Resprites Syndicate Commander Headset Chat Icon

### PR DESCRIPTION
**Pull request by Piesuu**
[game objects] [QoL] 
## About the PR 
I've noticed multiple Admins playing as Nanotrasen Commanders during admin gimmicks before, Previously only the Podwars commander headset was available for use (to the best of my knowledge), the new headset has a better icon and was made specially for gimmicks
The Syndicate Commander chat icon star was not symmetrical before, now it is, I resprited it as it was the base for the Nanotrasen Commander chat Icon

## Why's this needed? 
Better Admin Gimmicks, Icon symmetry

## Changelog 

```changelog
(u)Piesuu
(+)Nanotrasen Commanders are now entitled to a special headset, worthy of their rank.
```

### CODE REVIEW

**Comment by Piesuu** [comment]
This is my first PR adding actual features instead of correcting spelling mistakes, There is likely a mistake I made somewhere.
I'm not sure if the NT commander headset has a proper spawn path, there seems to be no standard

File: code/obj/item/device/radios/headsets.dm [hunk 1]
```diff
@@ -61,6 +61,21 @@
 	emp_act()
 		return //hax
 
+/obj/item/device/radio/headset/command/nt/commander
+	name = "\improper NT Commander's headset"
+	desc = "Issued to NanoTrasen Commanders, this radio headset can access several secure radio channels."
```
Comment (ZeWaka) [inline code/obj/item/device/radios/headsets.dm:66]: i would put this next to the syndicate headset probably?

Comment (Piesuu) [reply code/obj/item/device/radios/headsets.dm:66]: I put it next to the command headsets as It's part of the command category like the NT one, no trouble moving it next to the Syndicate ones if that's more fitting

Comment (Piesuu) [reply code/obj/item/device/radios/headsets.dm]: It's right above the Syndicate headset now

**Review by frawhst — APPROVED** [review]
`headset/command/nt/commander` lol

**Pull request merged**